LINUX/README

```diff
@@ -267,6 +267,17 @@ COMMON PROBLEMS
   In emulated netmap mode (i.e. with unpatched drivers) VLAN tags are never
   visible by the netmap application.
 
+* if you are using e1000 with netmap, you verify the result of the patch from
+  final-patches/vanilla--e1000--20620--31200 that was applied on
+  e1000/e1000_main.c. It was observed that when using older kernel version
+  netmap hook that needs to be applied on e1000_clean_rx_irq is actually 
+  applied on e1000_clean_jumbo_rx_irq. As a result it will end up in packets
+  going directly from NIC Rx Rings to Host stack.
+
```

Comment:
Can you remove all the text below and replace with something like "You should fix this issue by manually patching e1000/e1000_main.c to add the correct netmap hook in e1000_clean_rx_irq"?